src/Components/SmartComponents/Systems/SystemsTableToolbar.js

```diff
@@ -113,16 +96,9 @@ const SystemsTableToolbar = (props) => {
 
 SystemsTableToolbar.propTypes = {
     systems: dataShape,
-    params: propTypes.func,
-    setPage: propTypes.func,
-    setFilter: propTypes.func,
-    setOptOut: propTypes.func,
+    parameters: propTypes.func,
     selectedHosts: propTypes.array,
-    setSelectedHosts: propTypes.func,
-    doOptOut: propTypes.func,
-    onRefresh: propTypes.any,
-    parameters: propTypes.object,
-    handleSelect: propTypes.func,
+    methods: propTypes.func,
```

Comment:
you're passing an object of functions

` methods = {{
                                doOptOut,
                                apply,
                                handleSelect
                            }}`
it would be nice if you'll use shape, like 
``` 
methods: propTypes.shape({
         doOptOut: propTypes.func,
        apply: propTypes.func,
        handleSelect: propTypes.func
 })
```